.cirrus.yml

```diff
@@ -69,7 +69,7 @@ linux_task:
   install_script:
     - python -m pip install --retries 3 --upgrade pip
     - pip install --retries 3 -r requirements.txt
-    - python setup.py develop
+    - python -m pip install -e .
```

Comment:
hmm, I don't think we should make this editable

Reply:
I do not understand. `-e` means that package will use project structure as source root. So test will use same files which is installed as package. 

Maybe best solution is to use tox. Did You talk about this?

Reply:
ah ok nevermind then

Reply:
I like this change as is.